content/en/docs/release-oversight/pull-request-testing.md

```diff
@@ -0,0 +1,275 @@
+
+Examples when /payload blocking/informing should be used:
+  * Major Kube rebases, CoreOS updates, ovn/network upgrades.
+  * New invariant tests that could start failing payloads.
+
+Examples when /payload blocking/informing *should not* be used:
+  * To gather results for a specific job. (see /payload-job and /payload-aggregate below)
+  * Any low risk change.
+
+Any collaborator of the GitHub OpenShift organization can issue the command on a pull request to a branch of a repository of the organization that contributes to OpenShift official images:
+
+> `/payload <ocp_version> <ci|nightly> <informing|blocking>`
+
+For example, if `/payload 4.10 nightly informing` is issued on a PR, the robot will reply the list of the triggered jobs:
+
+![payload command](/payload-cmd.png)
+
+The jobs triggered by the command are determined by [OpenShift Release Controllers](/docs/getting-started/useful-links/#services).
+
+The linked page from [payload-tests portal](https://pr-payload-tests.ci.openshift.org/runs/) at the bottom of the comment shows the status of the payload testing and the details of those jobs.
+
+You can also invoke this command with multiple PRs:
+
+> `/payload-with-prs <ocp_version> <ci|nightly> <informing|blocking> <org/repo#number> [<org/repo#number ...]`
+
+### /payload-job
+
+A particular job or set of jobs can be triggered by `/payload-job`, such as
+
+> `/payload-job <periodic_ci_openshift_release_some_job> <periodic_ci_openshift_release_another_job>`
+
+This command can be repeated in one comment to launch multiple copies of the same job. (this differs from /payload-aggregate which will attempt Fisher's Exact test on the results)
+
+You can also invoke this command with multiple PRs:
+
+> `/payload-job-with-prs <periodic_ci_openshift_release_some_job> <org/repo#number> [<org/repo#number ...]`
+
+### /payload-aggregate
+
+It is also possible to perform aggregation on any of these jobs with /payload-aggregate. This launches the specified number of job runs, and attempts aggregation with Fisher's Exact.
+
+> `/payload-aggregate <periodic_ci_openshift_release_some_job> <aggregated_count>`
+
+You can also invoke this command with multiple PRs:
+
+> `/payload-aggregate-with-prs <periodic_ci_openshift_release_some_job> <aggregated_count> <org/repo#number> [<org/repo#number ...]`
+
+{{% alert title="NOTE" color="warning" %}}
+`/payload-with-prs`, `/payload-aggregate`, `/payload-aggregate-with-prs`, and `/payload-job-with-prs` only accept a single command per comment; additional commands need to be triggered with separate comments (not just separate lines).
+{{% /alert %}}
+
+### /payload-abort
+
+It is possible to quickly abort all running payload jobs for a specific PR. Simply comment `/payload-abort` on the PR to do so.
+This is particularly helpful when a large number of jobs have been launched.
+
+### Manually Submitting a `PRPQR`
```

Comment:
I think this is a poor thing to keep in this documentation, far too low level, not something anyone needs to do (I don't think I've ever done this), and a prime candidate for getting out of date long before anyone actually needs to use it.

Reply:
docs.ci has quite a lot of documentation for test platform to use (who owns the PRPQR CRD), and much of it gets into the internals of CI.  I think it's important it remains somewhere.

Reply:
There are a few features in PRPQR that are only accessible via manual creation of the CRD. It is not possible to have a GH comment based UI that handles all possible scenarios.